fuse_publishers/src/pose_2d_publisher.cpp

```diff
@@ -84,12 +85,14 @@ bool findPose(
   }
   catch (const std::exception& e)
   {
-    ROS_WARN_STREAM_THROTTLE(10.0, "Failed to find a pose at time " << stamp << ". Error" << e.what());
+    RCLCPP_WARN_STREAM_THROTTLE(rclcpp::get_logger("pose_2d_publisher"), rclcpp::Clock(), 10.0 * 1000,
```

Comment:
Any places where we don't have a logger, I'd recommend using the package name as the logger name. I think that will make it easier for users to know where messages are coming from.

Reply:
https://github.com/locusrobotics/fuse/pull/279/commits/56b274b6752572f8314dcaf32bea79c54ae213e5 !